Header header

uint32 num_person
COCO3d[] data